modules/op-creating-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -32,7 +44,48 @@ You can specify the parameters of your commit and URL by using dynamic, expandab
 * `{{git_auth_secret}}`: The secret name that is generated automatically with Git provider's token for checking out private repos.
 
 [discrete]
-.Matching an event to a pipeline run
+.Using the temporary GitHub App token for Github API operations
+
+You can use the temporary installation token generated by {pac} from GitHub App to access the GitHub API. The token value is stored in the temporary `{{git_auth_secret}}` dynamic variable generated for private repositories in the `git-provider-token` key.
```

Comment:
ISG asks for "that" to set off restrictive clauses. 

```suggestion
You can use the temporary installation token generated by {pac} from GitHub App to access the GitHub API. The token value is stored in the temporary `{{git_auth_secret}}` dynamic variable that is generated for private repositories in the `git-provider-token` key.
```

Reply:
this is tricky, I had to rewrite and will need to get QE approval.